src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let currentResult = 0;
+  let result = 0;
+
+  function adder(value) {
+    if (arguments.length !== 0) {
+      currentResult += value;
+
+      return adder;
+    }
+    result = currentResult;
```

Comment:
```suggestion
  let currentResult = 0;

  function adder(value) {
    if (arguments.length !== 0) {
      currentResult += value;

      return adder;
    }
    const result = currentResult;
```
You use `result` only here so it can be `const`